lib/rollbar/plugins/active_job.rb

```diff
@@ -1,35 +1,44 @@
-module Rollbar
-  # Report any uncaught errors in a job to Rollbar and reraise
-  module ActiveJob
-    def self.included(base)
-      base.send :rescue_from, Exception do |exception|
-        args = if self.class.respond_to?(:log_arguments?) && !self.class.log_arguments?
-                 arguments.map(&Rollbar::Scrubbers.method(:scrub_value))
-               else
-                 arguments
-               end
+Rollbar.plugins.define('active_job') do
+  dependency { !configuration.disable_monkey_patch }
+  dependency { !configuration.disable_action_mailer_monkey_patch }
 
-        Rollbar.error(exception,
-                      :job => self.class.name,
-                      :job_id => job_id,
-                      :use_exception_level_filters => true,
-                      :arguments => args)
-        raise exception
+  execute do
+    module Rollbar
+      # Report any uncaught errors in a job to Rollbar and reraise
+      module ActiveJob
+        def self.included(base)
+          base.send :rescue_from, Exception do |exception|
+            args = if self.class.respond_to?(:log_arguments?) && !self.class.log_arguments?
+                     arguments.map(&Rollbar::Scrubbers.method(:scrub_value))
```

Comment:
Hey @waltjones, I noticed that the rollbar update to 3.5.0 broke one of the mailers of our app and tracked it down here: `ActionMailer::Base` doesn't have an `arguments` method and `NameError: undefined local variable or method 'arguments' for #<YourMailerClass:0x0000000036ada8>>` here

I assume this was fixed on the tests here with the `attr_accessor :arguments` on the TestMailer, but that doesn't happen on a regular rails app

Reply:
@lucasmazza Fix is released in 3.5.1. https://github.com/rollbar/rollbar-gem/releases/tag/v3.5.1